src/bonnie/FINRA.java

```diff
@@ -0,0 +1,31 @@
+package src.bonnie;
+
+public class FINRA {
+
+    public static void main(String[] args) {
+
+        finra();
+
+    }
+
+    public static void finra() {
+
+        String result = "";
+
+        for (int i = 1; i <= 30; i++) {
```

Comment:
For the sake of good practice, and to make your code dynamic assign your condition to a variable.

Reply:
Thanks for pointing that out. I have updated the code to be dynamic. Thanks!